### DISCUSSION TITLE
Attributes are not applied to group field type

### DISCUSSION THREAD

**Discussion by samuel-clara** [solspace/craft-freeform#1477] [Questions & Answers]
### What happened?

Attributes added inside the builder to a group field on a form are not applied. I'm using Freeform's built-in render helpers.

### How can we reproduce this?

1. Add an attribute  with a class to the container of a Group field type
2. Class attribute is not rendered

### Environment

- Freeform Pro 5.5.5
- Craft 5.2.9

**Comment by kjmartens** [comment]
Sorry for the delay @samuel-clara,

I just tested this on my end, and it seems to work correctly.

Just to clarify, are you expecting these attributes to apply to the Group container and label, or the fields within the group? It will only apply itself to the Group container and label. 🙂
